analytics.js

```diff
@@ -529,7 +535,11 @@ class Analytics {
     } else {
       rudderElement.setProperty({});
     }
-
+    Object.keys(properties).forEach((property) => {
```

Comment:
add a util function and pass props/traits as parameter ?
also can this check be done at a central loc like in https://github.com/rudderlabs/rudder-sdk-js/blob/production/analytics.js#L638